lib/iex/lib/iex/info.ex

```diff
@@ -87,10 +87,11 @@ defimpl IEx.Info, for: List do
 
   defp info_char_list(char_list) do
     desc = """
-    This is a list of integers that is printed as a sequence of codepoints
+    This is a list of integers that is printed as a string of characters
```

Comment:
I think the word "string" here is confusing because of our double-quoted strings. Sequence is better.